packages/policy/src/cli.ts

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import meow from 'meow';
+import {Octokit} from '@octokit/rest';
+import {exportToBigQuery} from './export';
+import {getPolicy, GitHubRepo} from './policy';
+
+interface Flags {
```

Comment:
I like that we're landing on this CLI based approach for testing.

Reply:
Yeah this was super useful while writing the bot!  It did rely on a personal access token, so I don't know how I feel about that, but I was largely inspired by what you and @SurferJeffAtGoogle were doing in owl bot.